specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/deploymentScripts.json

```diff
@@ -752,6 +752,10 @@
             "modelAsString": true
           }
         },
+        "tenantId": {
```

Comment:
Info for other reviewers: The service has always returned the tenant ID. @lwang2016 is fixing a swagger accuracy issue.

Reply:
Correct. Thanks for the clarification @majastrz